xtractmime/_utils.py

```diff
@@ -46,14 +46,14 @@
 ]
```

Comment:
`MP25_RATES` and `MP3_RATES` should probably be tuples as well, for the same reason as `SAMPLE_RATES`.